xbmc/cores/VideoPlayer/DVDCodecs/Video/VAAPI.cpp

```diff
@@ -2006,7 +2006,11 @@ void COutput::InitCycle()
         ||  method == VS_INTERLACEMETHOD_RENDER_BOB)
     {
         if (method == VS_INTERLACEMETHOD_AUTO)
+#if defined(HAVE_VPP)
```

Comment:
I will remove all of them tonight.

Am 26.07.2016 10:34 vorm. schrieb "Rainer Hochecker" <
notifications@github.com>:

> In xbmc/cores/VideoPlayer/DVDCodecs/Video/VAAPI.cpp
> <https://github.com/xbmc/xbmc/pull/10170#discussion_r72208513>:
>
> > @@ -2006,7 +2006,11 @@ void COutput::InitCycle()
> >          ||  method == VS_INTERLACEMETHOD_RENDER_BOB)
> >      {
> >          if (method == VS_INTERLACEMETHOD_AUTO)
> > +#if defined(HAVE_VPP)
>
> if we want to consider every silly scenario that may come to user's minds,
> we need to introduce some thousands more of those ifdefs. Having this
> available in a build environment is a requirement now.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/xbmc/xbmc/pull/10170/files/2671f2557674bbc69565e8060d71a532c589320b#r72208513>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABCfHWLFxOSDO-oGStrDNJgCWITCFC2-ks5qZcaVgaJpZM4JUyTa>
> .
>